src/components/Map.js

```diff
@@ -1,20 +1,32 @@
 import React, { useContext, useState } from 'react';
 
 import GoogleMapReact from 'google-map-react';
+import { Dialog } from '@reach/dialog';
+import '@reach/dialog/styles.css';
```

Comment:
It's interesting, you can import your own Reach UI states. Nice.

Reply:
yeah, even though in our code, we're using our styles, it's still a requirement to import Reach UI styles😄